File: Authorization.Data/Entities/Org.cs
using System;
using System.Collections.Generic;

namespace Authorization.Data.Entities
{
    public class Org
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public bool IsDeleted { get; set; }

        public List<Item> Items { get; set; }
        public List<OrgUser> OrgUsers { get; set; }
        public List<User> OrgDefaultUsers { get; set; }
    }
}

File: Authorization.Web/ClientApp/src/app/models/api/org.ts
import { Item } from './item';
import { OrgUser } from './org-user';
import { User } from './user';

export interface Org {
  id: number;
  name: string;
  isDeleted: boolean;

  items: Item[];
  orgDefaultUsers: User[];
  orgUsers: OrgUser[];
}
